packages/network-controller/src/NetworkController.ts

```diff
@@ -145,12 +145,8 @@ const LOCALHOST_RPC_URL = 'http://localhost:8545';
 
 const name = 'NetworkController';
 
-type BlockTracker = any;
```

Comment:
Extracted these types to their own file so that they could be shared between this file and `create-network-client` more easily.